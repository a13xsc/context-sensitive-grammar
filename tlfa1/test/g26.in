({.,P,E,F,X,H,D,^,R,Y,3,B,T,I,S,A,K,J,O,@},{.,@,^,3},{(A,3@APHF),(J,RDJIDREDPDIXK),(J,e),(J,@.JIK.3D),(S,.EJDK@PO.S.OYFO^S)},J)
