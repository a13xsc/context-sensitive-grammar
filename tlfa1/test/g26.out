O
H
X
E
A
F
R
B
T
S
Y
I
K
P
D
